packages/node-cli/src/commands/file/import/ImportFilesAction.js

```diff
@@ -31,6 +32,7 @@ module.exports = class ImportFilesAction extends BaseAction {
 		super(options);
 
 		this._projectInfoService = new ProjectInfoService(this._projectFolder);
+		this._calledFromCompareFiles = true;
```

Comment:
why is the default set to true. Should not be false?

Reply:
Should be false. Updated.